atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -216,22 +220,35 @@ def manifest_list_entries_match(self, image, build_id):
                 return False
 
             arch = manifest['platform']['architecture']
+            if arch not in self.architectures:
+                self.log.warning('Requested platform %s is missing in manifest list', arch)
```

Comment:
we don't need this,
it is already checked in pull_base_image

Reply:
Removed, thanks